tools/packaging/guest-image/build_image.sh

```diff
@@ -47,6 +47,13 @@ build_initrd() {
 		kernel_version=${kernel_version#v}
 		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/cc-sev-kernel/builddir/kata-linux-${kernel_version}-${config_version}/lib/modules/${kernel_version}"
 		sudo -E PATH="$PATH" make rootfs ROOTFS_BUILD_DEST="${rootfs_build_dest}" KERNEL_MODULES_DIR="${module_dir}"
+	elif [[ "${AA_KBC:-}" == "offline_fs_kbc" ]] && [[ "${arch_target}" == "s390x" ]]; then
+		# IBM Z SE (secure execution) builds initrd with rootfs including common crypto kernel modules
+		config_version=$(get_config_version)
+		kernel_version="$(get_from_kata_deps "assets.kernel.se.version")"
+		kernel_version=${kernel_version#v}
+		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/cc-se-kernel/builddir/kata-linux-${kernel_version}-${config_version}/lib/modules/${kernel_version}"
+		sudo -E PATH="$PATH" make rootfs ROOTFS_BUILD_DEST="${rootfs_build_dest}" KERNEL_MODULES_DIR="${module_dir}"
```

Comment:
So just to clarify, we need these kernel modules to be loadable? It seems like it would be much simpler to have the builtin to the kernel.

Reply:
But, yes, I agree that the builtin approach looks much simpler. Thanks! :wink:

Reply:
@fidencio just to make sure that the built-in approach, Tobin mentioned, is something I had done before the current change which triggers the error message above?

> ```
> *** The present kernel configuration has modules disabled.
> *** To use the module feature, please run "make menuconfig" etc.
> *** to enable CONFIG_MODULES.
> ```

Reply:
https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kernel/configs/fragments/s390/crypto.conf isn't those built-in already?

Reply:
I meant the crypt modules from https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kernel/configs/fragments/common/confidential_containers/cryptsetup.conf which is included when `KATA_BUILD_CC` is set.

Reply:
So do you think including all the modules from https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kernel/configs/fragments/common/confidential_containers/cryptsetup.conf in https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kernel/configs/fragments/common/confidential_containers/cryptsetup.conf would be a simple solution? :wink: